app/Http/Controllers/TaskController.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Task;
+use Dotenv\Validator;
+use Illuminate\Auth\Events\Validated;
+use Illuminate\Contracts\Validation\Validator as ValidationValidator;
+use Illuminate\Http\Request;
+use Illuminate\Validation\Validator as IlluminateValidationValidator;
+use Nette\Utils\Validators;
+use PHPUnit\Util\Xml\Validator as XmlValidator;
+
+class TaskController extends Controller {
+    //public function task(){
+    //     return 
+    //    }
+
+    public function index() {
+        $tasks = Task::orderBy('created_at', 'asc')->get();
+
+        return view('tasks', [
+            'tasks' => $tasks
+        ]);
+    }
+
+    public function delete(){
+        function ($id) {
```

Comment:
remove `function ($id)` and pass id like `public function delete($id){`

Reply:
problem solved